website/docs/development/interfaces.md

```diff
@@ -18,10 +18,10 @@ As the APIObjectAdapter covers only API fields which are automatically generated
 
 The actuator implements the majority of the resource-specific functionality. It is split into several interfaces:
 
-* [`BaseResourceActuator`](../godoc/generic-interfaces/#BaseResourceActuator): methods required for all actuators
-* [`CreateResourceActuator`](../godoc/generic-interfaces/#CreateResourceActuator): methods required for creating or importing a resource
-* [`DeleteResourceActuator`](../godoc/generic-interfaces/#DeleteResourceActuator): methods required for deleting a resource
-* [`ReconcileResourceActuator`](../godoc/generic-interfaces/#ReconcileResourceActuator): methods required for reconciling a resource after creation
+* [`BaseResourceActuator`](godoc/generic-interfaces.md/#BaseResourceActuator): methods required for all actuators
+* [`CreateResourceActuator`](godoc/generic-interfaces.md/#CreateResourceActuator): methods required for creating or importing a resource
+* [`DeleteResourceActuator`](godoc/generic-interfaces.md#DeleteResourceActuator): methods required for deleting a resource
```

Comment:
You've removed a `/`. Was it intentional?

Reply:
Do you mean? `../` - yes

Reply:
No, you changed from `generic-interfaces/#DeleteResourceActuator` to `generic-interfaces#DeleteResourceActuator`

Reply:
AFAIR that wasn't intentional